tasks/task02/triangle.py

```diff
@@ -0,0 +1,50 @@
+# Shagov Aleksei
+import math
+
+
+def Submenu_Base_Height():
+	s = input("Enter your base and height: ")
+	parts = s.split(" ")
+	base = int(parts[0])
+	height = parts[1]
+	print(base, height)
+	area = 0.5 * float(base) * float(height)
```

Comment:
If we use only int value, that we need use this construction: `(base * height) / 2`. This will potentially correct the inaccuracy.